sim/core/target.go

```diff
@@ -329,11 +326,26 @@ func NewAttackTable(attacker *Unit, defender *Unit, weapon *Item) *AttackTable {
 		table.SpellCritSuppression = UnitLevelFloat64(defender.Level-attacker.Level, 0, 0, 0.003, 0.021)
 	} else {
 
+		levelDelta := 0.0004 * 5 * float64(defender.Level-attacker.Level)
+
 		table.BaseSpellMissChance = 0.05
-		table.BaseMissChance = UnitLevelFloat64(attacker.Level-defender.Level, 0.05, 0.048, 0.046, 0.044)
-		table.BaseBlockChance = UnitLevelFloat64(attacker.Level-defender.Level, 0.05, 0.048, 0.046, 0.044)
-		table.BaseDodgeChance = UnitLevelFloat64(attacker.Level-defender.Level, 0, -0.002, -0.004, -0.006)
-		table.BaseParryChance = UnitLevelFloat64(attacker.Level-defender.Level, 0, -0.002, -0.004, -0.006)
+
+		// Apply base Parry
+		if defender.PseudoStats.CanParry {
+			table.BaseParryChance = 0.05 + levelDelta
+		} else {
+			table.BaseParryChance = 0
+		}
+		// Apply base Block
+		if defender.PseudoStats.CanBlock {
+			table.BaseBlockChance = 0.05 + levelDelta
+		} else {
+			table.BaseBlockChance = 0
+		}
+
+		table.BaseMissChance = 0.05 + levelDelta
+		table.BaseDodgeChance = 0.05 + levelDelta
+		table.BaseCritChance = 0.05 + levelDelta
```

Comment:
BaseCritChance must be `0.05 - levelDelta`

Reply:
`levelDelta` will be negative if the attacker is higher level than the player target (such as a boss), so addition is correct here.

Reply:
ye, for e.g. a boss attacking player, `levelDelta` will be `-0.006`, (-0.6%) so Base{Miss,Dodge,Parry,Block}Chance will all be `0.044` (4.4%), but BaseCritChance of the boss should be `0.056` (5.6%)